scripts/git/cpplint.py

```diff
@@ -6660,8 +6660,8 @@ def FlagCxx11Features(filename, clean_lines, linenum, error):
     if include and include.group(1) in (
         "cfenv",
         "fenv.h",
-        "condition_variable",
-        "future",
+        # "condition_variable",
```

Comment:
why?

Reply:
the linter wasn't passing on Michael's code because he had headers "condition_variable" and "future", so I added it to the allowed header list